### PR TITLE
update contributors

### PR DESCRIPTION
**Pull request by weissi**

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -72,21 +76,26 @@ needs to be listed here.
...
+- Simon Evans <si@si.org>
 - Stuart Hinson <stuart.hinson@gmail.com>
 - Tanner <me@tanner.xyz>
 - Tiago Martinho <martinho@thoughtworks.com>
 - Tibor Bödecs <mail.tib@gmail.com>
 - Tim Condon <0xtimc@gmail.com>
 - Tim Kientzle <tkientzle@apple.com>
+- Tom Counsell <tamc@greenonblack.com>
 - Tomer Doron <tomerd@apple.com>
+- Trevör <adtrevor@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @adtrevor if you would like me to add a proper email address, please let me know which one.

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -49,13 +50,16 @@ needs to be listed here.
 - John Connolly <connoljo2@gmail.com>
 - John Holdsworth <github@johnholdsworth.com>
 - Jovan Milenkovic <jovan.milenkovic@htecgroup.com>
+- Julian Lettner <yln@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @yln if you would like me to add a proper email address, please let me know which one.

**Comment by yln** [comment]
julian.lettner@apple.com

File: CONTRIBUTORS.txt (hunk 2 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: done

**Pull request merged**